internal/future/assets/bindata.go

```diff
@@ -0,0 +1,367 @@
+// Code generated by go-bindata. DO NOT EDIT.
+// sources:
+// backend.swagger.json (16.416kB)
+// bindata.go (67.281kB)
```

Comment:
It seems like we are having a nested bindata file here.

I suggest organizing the bindata with the following structure:

```
.
├── frontend
|   └── swagger.json
├── backend
|   └── swagger.json
├── matchfunction
|   └── swagger.json
├── mmlogic
|   └── swagger.json
```

This gives us the ability to serve all the swagger files under the same file path (e.g localhost:51504/swagger/swagger.json)